tests/unit/config_tests.py

```diff
@@ -20,6 +20,8 @@ class TestConfigParser(unittest.TestCase):
         ('--keep-builddir', 'keep_builddir', True),
         ('--build-versions-repository-url=foo', 'build_versions_repository_url', 'foo'),
         ('--build-version=foo', 'build_version', 'foo'),
+        ('--mock-args=--enable-plugin=tmpfs --plugin-option=tmpfs:keep_mounted=True', 'mock_args',
```

Comment:
I think it's a bit misleading to use the options in this way inside a single string. The test will pass and there is nothing to do with the fact that there is two mock arguments there. AFAIK, this is not how argparse will return the values.